lib/bone.js

```diff
@@ -345,9 +345,14 @@ class Bone {
     const where = { [primaryKey]: this[primaryKey] };
     if (shardingKey) where[shardingKey] = this[shardingKey];
 
-    return Bone.update.call(Model, where, changes).then(count => {
-      if (count === 1) this.syncRaw(changes);
-      return count;
+    const { updatedAt, deletedAt } = Model.timestamps;
```

Comment:
`instance.update`  should independent from `Class.update`, that will be more easy to implement hooks